db/schema.rb

```diff
@@ -501,6 +502,9 @@
     t.integer "totchild"
     t.integer "hhtype"
     t.integer "hodate_check"
+    t.integer "extrabor_value_check"
+    t.integer "deposit_and_mortgage_value_check"
```

Comment:
These columns and the one on line 490 shouldn't be here? I think that's why the tests fail on remote

Reply:
where the hell have they come from?
I've really tied myself in knots with rebases today, not recommended.